q2_fmt/plugin_setup.py

```diff
@@ -360,4 +368,62 @@
     }
 )
 
+plugin.methods.register_function(
+    function=q2_fmt.peds_simulation,
+    inputs={'table': FeatureTable[Frequency | RelativeFrequency |
+                                  PresenceAbsence]},
+    parameters={'metadata': Metadata, 'time_column': Str,
```

Comment:
I like to list everything in the registration with one value per line, just for readability. This is just a suggestion, your call. 

```suggestion
    parameters={'metadata': Metadata, 
                'time_column': Str,
```